runtime/doc/api-ui-events.txt

```diff
@@ -846,6 +846,7 @@ must handle.
 		"lua_error"	Error in |:lua| code
 		"lua_print"	|print()| from |:lua| code
 		"rpc_error"	Error response from |rpcrequest()|
+		"progress"	Progress message emitted by |nvim_echo()|
```

Comment:
this list is almost sorted, can you move this up 1 line so that the list stays almost-sorted :)